src/electionguard-ui/ElectionGuard.UI/ViewModels/ManifestViewModel.cs

```diff
@@ -78,12 +78,13 @@ public ManifestViewModel(IServiceProvider serviceProvider, ManifestService manif
         for (ulong i = 0; i < value.CandidatesSize; i++)
         {
             var local = Manifest.GetCandidateAtIndex(i);
-            var party = Parties.FirstOrDefault(p => p.PartyId == local.PartyId);
-            if (party == null)
+            var party = Parties.FirstOrDefault(p => p.PartyId == local.PartyId) ?? new PartyDisplay(string.Empty, string.Empty, string.Empty);
+            var name = local.Name.GetTextAt(0).Value;
+            if(name == string.Empty)
             {
-                party = new PartyDisplay("", "", "");
+                name = "Writein";

```

Comment:
Set magic string as constant at top of either method or class

Reply:
changed to be a translated string.